File: TypeScript/Directory/all-documents-in-directory.ts
import * as fs from "fs"

function main(): void {
    // Get all the documents in a given directory
    console.log(getAllDocumentsInDirectory("assets/valid/j3U5uJY779L49q98MX86iFsxs2kY9ew3/"))
}

main()

// Get all the documents in a given directory
function getAllDocumentsInDirectory(directory: string): string[] {
    return fs.readdirSync(directory)
}

File: Dart/Directory/all_documents_in_directory.dart
import "dart:core";
import "dart:io";

void main() {
  // Get all the documents in a given directory.
  print(getEverythingFromDirectory("assets/valid/"));
}

// Get all the documents in a given directory
List<FileSystemEntity>? getEverythingFromDirectory(final String path) {
  return Directory(path).listSync().toList();
}
